proto/v1/cel-service.proto

```diff
@@ -0,0 +1,120 @@
+syntax = "proto3";
+
+package google.api.expr.v1;
+
+option cc_enable_arenas = true;
+option java_multiple_files = true;
+option java_outer_classname = "CelServiceProto";
+option java_package = "com.google.api.expr.v1";
+
+import "google/rpc/status.proto";
+import "proto/checked/v1/checked.proto";
+import "proto/v1/eval.proto";
+import "proto/v1/syntax.proto";
+
+// Access a CEL implementation from another process or machine.
+// A CEL implementation is decomposed as a parser, a static checker,
+// and an evaluator.  Every CEL implementation is expected to provide
+// a server for this API.  The API will be used for conformance testing,
+// utilities, and execution as a service.
+service CelService {
+  // Transforms CEL source text into a parsed representation.
+  rpc Parse (ParseRequest) returns (ParseResponse);
+
+  // Runs static checks on a parsed CEL representation and return
+  // an annotated representation, or a set of issues.
+  rpc Check (CheckRequest) returns (CheckResponse);
+
+  // Evaluates a parsed or annotation CEL representation given
+  // values of external bindings.
+  rpc Eval (EvalRequest) returns (EvalResponse);
+}
+
+// Warnings or errors in service execution are represented by
+// google.rpc.Status messages, with the following message
```

Comment:
Make sure to refer to message names with backticks: ``` `google.rpc.Status` ```

Reply:
Using square brackets instead, per https://cloud.google.com/apis/design/documentation#cross-references